scripts/run_latest_build.sh

```diff
@@ -41,65 +41,66 @@ HOSTNAME=${PUBLIC_IP}.nip.io
     origin_version=$(echo $ORIGIN_VERSION | tr -d v.)
     if (( $origin_version > $client_version )); then
-	echo "WARNING: Using client version: $version with cluster version: $ORIGIN_VERSION"
+        echo "WARNING: Using client version: $version with cluster version: $ORIGIN_VERSION"
     fi
 fi
 
-oc cluster up --image=${ORIGIN_IMAGE} \
-    --version=${ORIGIN_VERSION} \
-    --service-catalog=true \
-    --routing-suffix=${ROUTING_SUFFIX} \
-    --public-hostname=${HOSTNAME}
+if (( $client_version >= 3100 )); then
+    oc cluster up --image=${ORIGIN_IMAGE} \
+        --tag=${ORIGIN_VERSION} \
+        --enable=service-catalog,template-service-broker,router,registry,web-console \
+        --routing-suffix=${ROUTING_SUFFIX} \
+        --public-hostname=${HOSTNAME}
+else
+    oc cluster up --image=${ORIGIN_IMAGE} \
+        --version=${ORIGIN_VERSION} \
+        --service-catalog=true \
+        --routing-suffix=${ROUTING_SUFFIX} \
+        --public-hostname=${HOSTNAME}
+fi
+
+if [ "$?" -ne 0 ]; then
+    echo "Error starting cluster"
+    exit
+fi
 
 #
 # Logging in as system:admin so we can create a clusterrolebinding and
 # creating ansible-service-broker project
 #
 echo 'Logging in as "system:admin" to create broker resources...'
 oc login -u system:admin
-oc new-project ansible-service-broker
+oc new-project $BROKER_NAMESPACE
 
 #
-# A valid dockerhub username/password is required so the broker may
-# authenticate with dockerhub to:
-#
-#  1) inspect the available repositories in an organization
-#  2) read the manifest of each repository to determine metadata about
-#     the images
-#
-# This is how the Ansible Service Broker determines what content to
-# expose to the Service Catalog
-#
-# Note:  dockerhub API requirements require an authenticated user only,
-# the user does not need any special access beyond read access to the
-# organization.
-#
-# By default, the Ansible Service Broker will look at the
-# 'ansibleplaybookbundle' organization, this can be overridden with the
-# parameter DOCKERHUB_ORG being passed into the template.
-#
-TEMPLATE_URL=${TEMPLATE_URL:-"https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml"}
-DOCKERHUB_ORG=${DOCKERHUB_ORG:-"ansibleplaybookbundle"} # DocherHub org where APBs can be found, default 'ansibleplaybookbundle'
-ENABLE_BASIC_AUTH="false"
-VARS="-p BROKER_CA_CERT=$(oc get secret -n kube-service-catalog -o go-template='{{ range .items }}{{ if eq .type "kubernetes.io/service-account-token" }}{{ index .data "service-ca.crt" }}{{end}}{{"\n"}}{{end}}' | tail -n 1)"
-
-curl -s $TEMPLATE_URL \
-  | oc process \
-  -n ansible-service-broker \
-  -p DOCKERHUB_ORG="$DOCKERHUB_ORG" \
-  -p ENABLE_BASIC_AUTH="$ENABLE_BASIC_AUTH" \
-  -p NAMESPACE=ansible-service-broker \
-  $VARS -f - | oc create -f -
+# Use the automation-broker-apb to deploy the broker
+oc create serviceaccount $APB_NAME --namespace $BROKER_NAMESPACE
```

Comment:
Same question as below re: do we need the service account

Reply:
I actually was smart and delete that stuff in this script but forgot to add that to the k8s version.

Ref: https://github.com/openshift/ansible-service-broker/pull/891/files#diff-34011cced263e7ac1afc542ee01bf933R101